### PR TITLE
Development

### PR DESCRIPTION
**Pull request by asimzafar202**

### CODE REVIEW

File: hello.sh [hunk 1]
```diff
@@ -1,2 +1,8 @@
 #!/bin/bash
 
+echo "Hello Students"
+
+#i love lambos
+
+echo "anything" 
+vdfhbfgbdfghetr
```
Comment (asimzafar202) [inline hello.sh]: yo this doesnt look right

Comment (noahhaque) [inline hello.sh]: comment

**Review by noahhaque — CHANGES_REQUESTED** [review]
look at my comment

**Review by Noizebomb — APPROVED** [review]
looks good

**Pull request merged**